src/Component-Routes/MakeNote/MakeNote.jsx

```diff
@@ -0,0 +1,66 @@
+import './MakeNote.css';
+import { useState } from 'react';
+import { addDoc, collection } from '../../firebase-store/firebase-imports';
+import { auth,db } from '../../firebase-store/firebaseKeys';
+import { useNavigate } from 'react-router-dom';
+import Header from '../Layout-component/Header';
+
+function MakeNote({handleExit, UserInfo}) {
+
+const [ title, setTitle ] = useState('')
+const [ note, setNote ] = useState('')
```

Comment:
Esta raro No?? tener 2 estados para una misma nota tal vez pudo quedar así:
```suggestion
const [ note, setNote ] = useState({
title:'',
text:''
})
```